swap_meet/vendor.py

```diff
@@ -1,2 +1,83 @@
+from .item import Item
+
+
 class Vendor:
-    pass
+    def __init__(self, inventory=None):
+        if inventory:
+            self.inventory = inventory
+        else:
+            self.inventory = []
+
+    def add(self, item):
+        self.inventory.append(item)
+        return item
+
+    def remove(self, item):
+        if item in self.inventory:
+            self.inventory.remove(item)
+            return item
+        else:
+            return False
+
+    def get_by_category(self, category):
+        items_by_category = []
+        for item in self.inventory:
+            if item.category == category:
+                items_by_category.append(item)
+
+        return items_by_category
```

Comment:
We can refactor this function by using list comprehension:

```suggestion
    def get_by_category(self, category):
        items_by_category = [ item for item in self.inventory if item.category == category ]
        return items_by_category
```